language/move-lang/README.md

```diff
@@ -17,15 +17,44 @@ Move source language is an ergonomic language for writing Modules and Scripts th
+
+* **Stricter than Bytecode** The source language often adds additional level of restrictions. At an expression level, this means no arbitrary manipulation of the stack (only can do so through expressions), and no dead code or unused effects. At a module level this could mean additional warnings for unused types or un-invocable functions. And will also mean mean adding integration for formal verification.
+
+### Secondary Values
+
+* **Pathway of Learning** Syntax choices and error messages are intended to give a natural flow of learning. For example, some of the choices around expression syntax could be made more familiar, but they would hurt the plug-n-play feeling of the expression based syntax, which might hurt developing a deeper understanding of the language.
+
+* **Aiding Common Community Patterns** As Move becomes more heavily used, common patterns for modules are likely to appear. Move might add new language features to make these patterns easier, clearer, or safer. But not if it violates some other key design goal/value of the language.
+
+* **Careful Optimizations** Optimizations are an important developer tool, as they let a programmer write code in a more natural way. However, all of the optimizations performed must be semantic preserving, to prevent any catastrophic exploits or errors form occurring in optimized code. That being said, it is not the primary goal of the Move source language to produce *heavily* optimized code, but it is a nice feature to have.
+
+### Non Values
+
+* **Heavy Abstractions** The move source language does not intend to hide the details of the Move bytecode, this ranges from everything of references to global storage. There might be some abstractions that make interacting with these items easier, but they should always be available in Move at their most basic (bytecode equivalent) level. This does not mean that conveniences such as easy field access or implicit freezing are against the core set of values, but only that conveniences should not be ambiguous or opaque in how they interact at the bytecode level. Note though, this does not preclude the addition of features to the lanugage, such as access modifiers that that translate to compiler codegenerated dynamic checks. It is just that it is not an active goal of the language to add on heavy abstractions just for the sake of obscuring bytecode design choices.
```

Comment:
You should capitalize "move" in the first sentence here, and the last part of that first sentence does not parse. I don't think "codegenerated" is a word (at least not one that we should use here). You can just say "compiler generated".